content/en/blog/_posts/2020-11-02-remembering-dan-kohn.md

```diff
@@ -0,0 +1,16 @@
+---
+layout: blog
+title: "Remembering Dan Kohn"
+date: 2020-11-02
+slug: remembering-dan-kohn
+---
+
+**Author**: The Kubernetes Steering Committee
+
+Dan Kohn was instrumental in getting Kubernetes and CNCF community to where it is today. He shared our values, motivations, enthusiasm, community spirit and helped the Kubernetes community to become the best that it could be. He loved getting people together to solve problems big and small. He enabled people and has served to launch many a career of folks in the community.
```

Comment:
I think the last sentence reads awkward.

How about the following:

"He enabled people to grow their individual scope in the community which often helped launch their career in open source software."